app/schema/api/v4/models.rb

```diff
@@ -254,7 +254,8 @@ def teleconsultation
             properties: {
               requester_id: {"$ref" => "#/definitions/uuid"},
               facility_id: {"$ref" => "#/definitions/uuid"},
-              requested_at: {"$ref" => "#/definitions/timestamp"}
+              requested_at: {"$ref" => "#/definitions/timestamp"},
+              request_completed: {type: [:string, "null"], enum: Teleconsultation.request_completeds.keys << nil}
```

Comment:
Lets add 
```
ActiveSupport::Inflector.inflections do |inflect|
  inflect.irregular "request_completed", "requests_completed"
end
```
to config/initializers/inflections.rb

That way we can use `Teleconsultation.requests_completed` instead of `Teleconsultation.request_completeds`

Reply:
`requests_completed` sounds like a scope (of completed requests), shall we let this be since it's only going to be used in one place?